chall03/vde-dios.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import sys
+
+
+def snail(arg):
+    top = len(arg[1])
+    ini = 1
+    while (ini <= top):
+        i = ini
+        while (i <= top):
+            print(arg[ini][i - 1] if (ini is 1 and i is ini) else ", %s" % arg[ini][i - 1], end = '')
+            i += 1
+        i = ini + 1
+        while (i < top):
+            print(", %s" % arg[i][top - 1], end = '')
+            i += 1
+        if (ini < top):
+            i = top
+            while ( i >= ini):
+                print(", %s" % arg[top][i - 1], end = '')
+                i -= 1
+            i = top - 1
+            while ( i > ini):
+                print(", %s" % arg[i][ini - 1], end = '')
+                i -= 1
+        ini += 1
+        top -= 1
```

Comment:
This looks a lot like C. Check out some of your colleague's code for more "pythonic" ways to solve it (The ones who didn't cheat).